war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -1282,6 +1282,20 @@ var createPluginSetupWizard = function (appendTarget) {
   // do this so the page isn't blank while doing connectivity checks and other downloads
   setPanel(loadingPanel);
 
+  /* exported defaultUpdateSiteId */
+  var defaultUpdateSiteId = function () {
+    var defaultSiteId = document
+      .querySelector("#default-site-id")
+      .getAttribute("data-default-update-site-id");
+    return defaultSiteId ? defaultSiteId.replace("'", "") : "default";
+  };
```

Comment:
```suggestion
  var defaultUpdateSiteId = (function () {
    var defaultSiteId = document
      .querySelector("#default-site-id")
      .getAttribute("data-default-update-site-id");
    return defaultSiteId ? defaultSiteId.replace("'", "") : "default";
  })();
```
Perhaps IIFE syntax would work here? Haven't tested though.

Reply:
I did the changes you mentioned above and the ran `mvn clean verify -DskipTests` got successfully build message.
```
[INFO] Jenkins main module ................................ SUCCESS [  4.399 s]
[INFO] Jenkins BOM ........................................ SUCCESS [  0.291 s]
[INFO] Internal SPI for WebSocket ......................... SUCCESS [  3.260 s]
[INFO] Jetty 10 implementation for WebSocket .............. SUCCESS [  2.961 s]
[INFO] Jenkins cli ........................................ SUCCESS [  7.056 s]
[INFO] Jenkins core ....................................... SUCCESS [01:10 min]
[INFO] Jenkins war ........................................ SUCCESS [ 46.671 s]
[INFO] Tests for Jenkins core ............................. SUCCESS [ 18.275 s]
[INFO] Jenkins coverage ................................... SUCCESS [  0.465 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```
If anything else needs to be done in order to test this change please do let me know. If not, then give a thumbs up to this message so I can commit the changes.

Reply:
To test it check whether it actually works in runtime. I'd go through setup wizard steps a few times with console open and observe whether something unusual pops up.

Reply:
@yaroslavafenkin should I install all suggested plugins or is it not needed? Or should I test with both suggested select and not selecting anything?

Reply:
Installed plugins don't matter here.